src/RNTupleReader.cc

```diff
@@ -104,9 +104,16 @@ unsigned RNTupleReader::getEntries(const std::string& name) {
     m_readerEntries[name].push_back(0);
     for (const auto& filename : m_filenames) {
       try {
+#if ROOT_VERSION_CODE >= ROOT_VERSION(6, 36, 0)
+        ROOT::RNTupleDescriptor::RCreateModelOptions options;
```

Comment:
This isn't available earlier in some `Experimental` namespace? Or does it in any case not matter because with earlier ROOT versions this does not yet work in any case?